### PR TITLE
MO sos updates

### PR DESCRIPTION
**Pull request by hdansou**

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-535821e564/how-to/).

**Comment by catlook** [comment]
Honsou, please update the audit_date to today in the meta section at the top, then I get this merged in :)  Thanks for your help with this!

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c833f9a163/how-to/)

File: content/managed-operations/cloud-servers-with-managed-operations-support-for-windows.md [hunk 1]
```diff
@@ -113,17 +112,17 @@ Linux](/how-to/cloud-servers-with-managed-operations-support-for-linux).
 
 ### Other packages
 
-<span>While we do not support all technologies, we do offer reasonable
-endeavor support which extends our support into offering alternative
-solutions. The reasonable endeavor support can include help from Rackspace partners and other
+<span>While we do not support all technologies, we do offer reasonable-endeavor
```
Comment (DonNateR) [inline content/managed-operations/cloud-servers-with-managed-operations-support-for-windows.md]: Remove the span tag here.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-d4b4382b7f/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-0cc69987b2/how-to/). :bow:

**Comment by catlook** [comment]
@kmsholcomb

**Comment by kmsholcomb** [comment]
Hi @catlook. See the attached Word doc (or the PDF copy of it) for my suggested edits. Most are nitpicky! I did add some intro sentences for various sections, to provide a bit more context and align better with the article's Linux counterpart (see https://support.rackspace.com/how-to/cloud-servers-with-managed-operations-support-for-linux/). Please let me know if you have any questions. Thanks!

[Cloud Servers with Managed Operations support for Windows_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/717775/Cloud.Servers.with.Managed.Operations.support.for.Windows_editsKH.pdf)
[Cloud Servers with Managed Operations support for Windows_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/717776/Cloud.Servers.with.Managed.Operations.support.for.Windows_editsKH.docx)

**Comment by catlook** [comment]
Thanks, Kelly!

-- Cat Lookabaugh, Sr. Information Developer
Rackspace Hosting – the #1 Managed Cloud Company
Info Dev Team      cat.lookabaugh@rackspace.com
Office: (210) 312-8193   Cell: (210) 364-1060

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3b6ec89a45/how-to/)

**Pull request merged**